### ISSUE TITLE
Version 15082 "About" page not loading in Firefox browser.

### ISSUE THREAD

**Issue by FireWizard52** [domoticz/domoticz#5603]
Hi,

I installed the latest beta 15082 and noticed that clicking the "About" page in Settings, causes an issue in Firefox, version 110.0 ( 64 bit).
All other pages are loading and responsive, but the "About" page causes Firefox to hang. You can terminate it, when it indicates "Not responding"  It happens on two different instances of Domoticz, but not on all Firefox browsers. On another machine (same Firefox) it is okay. Chrome browser is okay.

Has there been any change regarding this page?

Regards

**Comment by gizmocuz** [comment]
2 months ago the active user was added.
I don't suspect this is the issue

https://github.com/domoticz/domoticz/commits/development/www/views/about.html

No other changes

**Comment by gizmocuz** [comment]
Did you update Firefox? Using Edge browser here... no issues and lighting fast!

**Comment by FireWizard52** [comment]
Thanks @gizmocuz  for your response.

No, It has worked since the active user has been added before and so I do not expect that this is the issue.

Yes, and I had an update on my Desktop today and Firefox has been updated. 
Now i know, that there has been no changes and I believe I have to look at Firefox.

I use Chrome as well without any issues. Perhaps I should switch to Chrome.

We do not spend more time on this issue and close it.

Regards

**Comment by gizmocuz** [comment]
My tip, instead of Chrome use Edge 👌 (Based on Chrome and same extensions)

**Comment by FireWizard52** [comment]
Thanks for the tip, but it will not work, as I used Windows private for the last time in the past century and since my retirement 6 years ago I completely left Windows behind me. 

I know it is available for Linux, but their is no reason to install it, as long browsers, such as Firefox, Opera, Chrome, etc. exist

I started to use Linux with KDE about 1997 and since then I never left it. 

But anyhow, thanks for the response.

Regards
